apisix/cli/ops.lua

```diff
@@ -427,7 +428,7 @@ Please modify "admin_key" in conf/config.yaml .
                 local ip = value.ip
                 local port = value.port
                 local enable_ipv6 = false
-                local enable_http2 = value.enable_http2
+                local enable_http2 = value.enable_http2         
 
```

Comment:
pls revert this line, the `tab` is useless

Reply:
PTAL, this should be fixed by https://github.com/apache/apisix/pull/10887/commits/f979676d06b1337e15481882cb762d36e449f232.